src/DependencyInjection/TacticianExtension.php

```diff
@@ -13,7 +13,7 @@ class TacticianExtension extends ConfigurableExtension
     /**
      * Configures the passed container according to the merged configuration.
      *
-     * @param array $mergedConfig
+     * @param array            $mergedConfig
```

Comment:
I'm not a huge fan either but I checked this against a few other League packages and they all use this type of alignment, so we should probably try to be consistent where we can.